constant-time/src/compare.rs

```diff
@@ -2,14 +2,28 @@
 
 use core::ptr;
 
-/// Little endian memcmp version of quinier/memsec
-/// https://github.com/quininer/memsec/blob/bbc647967ff6d20d6dccf1c85f5d9037fcadd3b0/src/lib.rs#L30
+/// Little endian memcmp version of [quinier/memsec](https://github.com/quininer/memsec/blob/bbc647967ff6d20d6dccf1c85f5d9037fcadd3b0/src/lib.rs#L30)
 ///
 /// # Panic & Safety
 ///
 /// Both input arrays must be at least of the indicated length.
 ///
 /// See [std::ptr::read_volatile] on safety.
+///
+/// # Examples
+/// ```no_run
```

Comment:
I think this fails to compile since it does not find `memcmp_le`. I think you should either import it via something like `use rosenpass_constant_time::memcmp_le` or replace `no_run` with `ignore`.

Reply:
I am not able to import `memcmp_le` because the module `compare` is private.

I can move the test into a unit-test instread and add the `ignore` attribute instread

Reply:
@aminfa You can also just make the compare module public if that is appropriate. We generally have an "everything is public but no API is stable" policy at the moment.

Reply:
Ok, I exported memcmp_le as a public item and removed the `ignore` doc-test attribute